runelite-client/src/main/java/net/runelite/client/plugins/varrockmuseum/VarrockMuseumPlugin.java

```diff
@@ -0,0 +1,220 @@
+package net.runelite.client.plugins.varrockmuseum;
+
+import com.google.common.collect.ImmutableSet;
+import com.google.common.eventbus.Subscribe;
+import net.runelite.api.Client;
+import net.runelite.api.GameState;
+import net.runelite.api.events.GameStateChanged;
+import net.runelite.api.events.WidgetLoaded;
+import net.runelite.api.widgets.Widget;
+import net.runelite.api.widgets.WidgetID;
+import net.runelite.api.widgets.WidgetInfo;
+import net.runelite.client.plugins.Plugin;
+import net.runelite.client.plugins.PluginDescriptor;
+import net.runelite.client.ui.overlay.OverlayManager;
+
+import javax.inject.Inject;
+import java.util.HashMap;
+import java.util.Set;
+
+@PluginDescriptor(
+	name = "Varrock Museum Quiz"
+)
+public class VarrockMuseumPlugin extends Plugin
+{
+	private static final Set<Integer> VARROCK_MUSEUM_MAP_REGIONS = ImmutableSet.of(6732, 6733, 6734, 6988, 6989, 6990, 7244, 7245, 7246);
+	private static boolean inMuseum = false;
+	private HashMap<String, String> questions = new HashMap<>();
+	private WidgetInfo correctAnswerID;
+
+	@Inject
+	private Client client;
+
+	@Inject
+	private VarrockMuseumOverlay overlay;
+
+	@Inject
+	private OverlayManager overlayManager;
+	@Override
+	protected void startUp () throws Exception
+	{
+		questions.put("How does a lizard regulate body heat?", "Sunlight.");
```

Comment:
This should be probably enum or some other sort of constant. But, preferrably enum

Reply:
I'm not entirely sure how the enums work which is why I used a hashmap but yes I agree, it probably should be something else